src/seedu/addressbook/commands/ListCommand.java

```diff
@@ -5,6 +5,7 @@
 import java.util.List;
 
 
+
```

Comment:
Extra blank line. Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.